web/src/ui/Visualizer/renderers/GraphCard.js

```diff
@@ -47,6 +48,9 @@ const GraphCard = (data, type) => {
           <div className="title">
             {data.title.replace(/"/gi, '\'')}
           </div>
+          <div className="estimated-time">
```

Comment:
Should be in a « if not empty », nope?

Reply:
Basically I had the idea to display undefined in the time when it did not exist, but after consideration it pollutes the cards without giving information so I fixed to display the time only if it exists!